samples/Samples.Server/Startup.cs

```diff
@@ -41,20 +43,17 @@ public Startup(IConfiguration configuration, IWebHostEnvironment environment)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-#if NETCOREAPP3_0
-            // Workaround until GraphQL can swap off Newtonsoft.Json and onto the new MS one.
-            // Depending on whether you're using IIS or Kestrel, the code required is different
-            // See: https://github.com/graphql-dotnet/graphql-dotnet/issues/1116
-            services.Configure<KestrelServerOptions>(options =>
-            {
-                options.AllowSynchronousIO = true;
-            });
-            services.Configure<IISServerOptions>(options =>
-            {
-                options.AllowSynchronousIO = true;
-            });
+#if NETCOREAPP2_2
+            services.AddSingleton<IGraphQLRequestDeserializer>(p =>
+                new GraphQL.Server.Serialization.NewtonsoftJson.GraphQLRequestDeserializer(settings => { }));
+#else
+            services.AddSingleton<IGraphQLRequestDeserializer>(p =>
+                new GraphQL.Server.Serialization.SystemTextJson.GraphQLRequestDeserializer(settings => { }));
 #endif
 
+            // TODO: Toggle use GraphQL.NewtonsoftJson.DocumentWriter or GraphQL.SystemTextJson.DocumentWriter once that PR over there is done
```

Comment:
As we talked earlier - by command line switch from launchSettings.

Reply:
Will deal with this in a later PR, as this PR can go out before the one over there